tools/migrate-to-armonia.sh

```
@@ -0,0 +1,459 @@
+
+  echo ""
+  echo "  Found paths in VS Code settings that may be stale after migration:"
+  echo ""
+
+  # Build the mapping and show it
+  local -a sed_args=()
+  for src_dir in "${REPO_SOURCES[@]}"; do
+    if grep -q "$src_dir" "$settings_file" 2>/dev/null; then
+      # Map the old source dir to armonia/repos (the move destination)
+      local escaped_src escaped_dest
+      escaped_src=$(printf '%s\n' "$src_dir" | sed 's/[&/\]/\\&/g')
```

Shellcheck (0.11.0)

[warning] escaped_src appears unused. Verify use (or export if used externally).

(SC2034)